superset/db_engine_specs/hive.py

```diff
@@ -128,33 +134,19 @@ def convert_to_hive_type(col_type: str) -> str:
                 "No upload bucket specified. You can specify one in the config file."
             )
 
-        table_name = form.name.data
-        schema_name = form.schema.data
-
-        if config["UPLOADED_CSV_HIVE_NAMESPACE"]:
-            if "." in table_name or schema_name:
-                raise Exception(
-                    "You can't specify a namespace. "
-                    "All tables will be uploaded to the `{}` namespace".format(
-                        config["HIVE_NAMESPACE"]
-                    )
-                )
-            full_table_name = "{}.{}".format(
-                config["UPLOADED_CSV_HIVE_NAMESPACE"], table_name
+        if "." in table_name and schema_name:
```

Comment:
Is this coming from a form? If so it would be best to add the validation check there.